app/javascript/mastodon/features/compose/components/compose_form.js

```diff
@@ -282,7 +286,6 @@ class ComposeForm extends ImmutablePureComponent {
             <Button
               type="submit"
```

Comment:
Linting didn’t seem to have triggered on previous PR. This should fix it:

```suggestion
              type='submit'
```